github-orgs/kubeflow/org.yaml

```diff
@@ -12,7 +12,7 @@ orgs:
         company: ""
         default_repository_permission: read
         description: Kubeflow is an open, community driven project to make it easy to deploy
-          and manage an ML stack on Kubernetes
+          and manage a ML stack on Kubernetes
```

Comment:
@sarahmaddox Can you just confirm it should be "a" not "an"

Reply:
It should be "an" because you'd pronounce the phrase as "an ML stack" not "a machine learning stack". :)

Reply:
Well, it sounded at bit strange to me, so I thought it should be 'a machine learning stack'. But it seems I've learned another thing for today 😄 

Just for my own curiosity (non-native English speaker), how is this different from `a machine learning algorithm` (used [here](https://www.dictionary.com/browse/machine-learning) and [here](https://en.wikipedia.org/wiki/Machine_learning#Applications)) or `a machine learning system` (used [here](https://www.linkedin.com/pulse/pros-cons-two-approaches-machine-learning-grammar-engineering-wei-li/)).

Or from the verb `a machine` [here](https://www.macmillandictionary.com/dictionary/british/machine_2)

Reply:
I've gone ahead and closed the PR. Feel free to continue the discussion.

Grammar was never my strong point so I don't have any insight on why one is more correct.